iptables/iptables.go

```diff
@@ -347,6 +347,34 @@ func existsRaw(table Table, chain string, rule ...string) bool {
 
 // Raw calls 'iptables' system command, passing supplied arguments.
 func Raw(args ...string) ([]byte, error) {
+	// TODO: Do we need this change ?
+	// check if the rule exists
+	var (
+		doCheck bool
```

Comment:
@aboch This is not need any more since we will remove the iptables during bridge driver initialization in https://github.com/docker/libnetwork/blob/master/drivers/bridge/bridge.go#L381